go.mod

```diff
@@ -49,3 +49,5 @@ require (
 )
 
 replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v0.9.2
+
+replace knative.dev/pkg => ../pkg
```

Comment:
/hold
```suggestion
# DO NOT SUBMIT
replace knative.dev/pkg => ../pkg
```

Reply:
I know I know, it was for testing purpose :smile: Once https://github.com/knative/pkg/pull/2127 is merged, I will update here accordingly